python/mxnet/symbol/symbol.py

```diff
@@ -745,6 +747,25 @@ def list_outputs(self):
             self.handle, ctypes.byref(size), ctypes.byref(sarr)))
         return [py_str(sarr[i]) for i in range(size.value)]
 
+    def output_count(self):
```

Comment:
Isn't this just `__len__`?

Reply:
I suppose one could use that convention. I can make it that.